Data/3_0/Skills/act_int.lua

```diff
@@ -5261,6 +5345,12 @@ skills["KineticBolt"] = {
 		["active_skill_additive_spell_damage_modifiers_apply_to_attack_damage_at_%_value"] = {
 			flag("ImprovedSpellDamageAppliesToAttacks"),
 		},
+		["cast_speed_+%_applies_to_attack_speed_at_%_of_original_value"] = {
```

Comment:
This relies on this PR (https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/1364) being merged, so use the same format as the one in the PR